packages/kbn-guided-onboarding/src/components/landing_page/guide/guide_cards.constants.tsx

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { ReactNode } from 'react';
+import { GuideId } from '../../../..';
+import { GuideCardSolutions } from './guide_cards';
+
+export interface GuideCardConstants {
+  solution: GuideCardSolutions;
+  title: string | ReactNode;
+  // if present, guideId indicates which guide is opened when clicking the card
+  guideId?: GuideId;
+  // if present, navigateTo indicates where the user will be redirected, when clicking the card
+  navigateTo?: {
+    appId: string;
+    path?: string;
+  };
+  // duplicate the telemetry id from the guide config to not load the config from the endpoint
+  // this might change if we decide to use the guide config for the cards
+  // see this issue https://github.com/elastic/kibana/issues/146672
+  telemetryId: string;
+  order: number;
+  icon: string;
+  // the guide will open a specific modal ESApiModal
+  openEndpointModal?: boolean;
+}
+
+export const guideCards: GuideCardConstants[] = [
+  {
+    solution: 'search',
+    icon: 'pivot',
+    title: i18n.translate('guidedOnboardingPackage.gettingStarted.cards.elasticsearchApi.title', {
+      defaultMessage: 'Connect to the Elasticsearch API',
```

Comment:
Were you able to confirm whether this should appear for all filters?

Reply:
It should not appear for observability or security yet. Search has confirmed they do want the modal cc @kevinsweet